internal/internal_workers_test.go

```diff
@@ -101,7 +101,7 @@ func (s *WorkersTestSuite) TestActivityWorker() {
 	hostEnv := getHostEnvironment()
 	hostEnv.addActivity(a.ActivityType().Name, a)
 	activityWorker := newActivityWorker(
-		s.service, domain, executionParameters, overrides, hostEnv,
+		s.service, domain, executionParameters, overrides, hostEnv, make(chan struct{}),
```

Comment:
We have to add ability to test the worker shutdown notification through ActivityTestEnvironment.

Reply:
Will do in https://github.com/uber-go/cadence-client/pull/677/